GiEnJul/ClientApp/src/routes/registerGiver/Confirmation.tsx

```diff
@@ -2,11 +2,19 @@ import * as React from 'react';
 import { Route } from 'react-router-dom';
 import { Button } from 'reactstrap';
 
-const Confirmation = () => {
+type Props = {
+    values: { location?: string; fullname?: string; email?: string; phoneNumber?: number; maxRecievers?: number; familyType?: string }
+}
+
+const Confirmation: React.FC<Props> = ({ values }) => {
+    console.log(values.fullname)
+    console.log(values.email)
+
+
     return (
         <div>
-            <h1>Takk for at du gir en jul</h1>
-            <p>Vi har nå registret deg som giver til årets Gi en Jul. vi har sendt en  oppsummering av din registrering i til Ola.Normann@gmail.com. Når det nermer seg jul vil vi sende deg mer informasjon. </p>
+            <h1>Takk for at du gir en jul {values.fullname}</h1>
+            <p>Vi har nå registret deg som giver til årets Gi en Jul og har sendt en  oppsummering av din registrering i til  { values.email}. Når det nermer seg jul vil vi sende deg mer informasjon. </p>
```

Comment:
Just some typos. Text should probably be seperated to a .resx file at a later time, as Ronny has mentioned.
```suggestion
            <p>Vi har nå registrert deg som giver til årets Gi en Jul og har sendt en  oppsummering av din registrering i til  { values.email}. Når det nermer seg jul vil vi sende deg mer informasjon. </p>
```

Reply:
Thanks, agreed! I have an idea of how to solve the editing of text via admin :D